tests/unit/test_reddit.py

```diff
@@ -39,6 +41,15 @@ def test_info__invalid_param(self):
 
         assert str(excinfo.value) == 'fullnames must be a list'
 
+    def test_live_info__valid_param(self):
```

Comment:
I'm going to merge as is, but these tests should probably be relocated to another unit test file.